custom_components/climate_group/climate.py

```diff
@@ -320,8 +320,16 @@ def async_update_group_state(self) -> None:
         # so that we don't break in the future when a new feature is added.
         self._attr_supported_features &= SUPPORT_FLAGS
 
+        self._enable_turn_on_off_backwards_compatibility = False
+
         _LOGGER.debug("State update complete")
 
+    async def async_turn_off(self) -> None:
+        await self.async_set_hvac_mode(HVACMode.OFF)
+
+    async def async_turn_on(self) -> None:
+        await self.async_set_hvac_mode(HVACMode.HEAT)
```

Comment:
This is the culprit.  Why assume on means heat mode?

Reply:
@arcsur I'm running in to this too, is there a way around this or do I need to edit the climate.py file?

Reply:
OK I changed `await self.async_set_hvac_mode(HVACMode.HEAT)` to `await self.async_set_hvac_mode(HVACMode.HEAT_COOL)` and it's working for my needs again.

Reply:
@alexeiw123 Remove or comment out the two functions `async_turn_on` and `async_turn_off`.  The default core climate methods will be used instead which does not force a startup mode but instead uses whatever the last state that HA remembers is.

Reply:
@bjrnptrsn Can the above change be considered or if you don't have time, just rollback this PR as it is causing issues by making invalid assumptions as to the turn on mode.

Reply:
I have finally been able to work on the problem and think that the behaviour of the 'turn on' command is now better implemented in the current version 1.0.7.

Reply:
@bjrnptrsn Thank you for working on this.  I think your solution covers the use cases.